config/environments/production.rb

```diff
@@ -131,7 +131,7 @@
   config.assets.precompile += javascript.map{|js| js + ".js"} + stylesheets + %w(background-size.htc)
 
   # Force all access to the app over SSL, use Strict-Transport-Security, and use secure cookies.
-  # config.force_ssl = true
+  config.force_ssl = true if SETTINGS[:require_ssl]
```

Comment:
That seems a bit dangerous if the file is misconfigured, `'false'` gets evaluated to true.